data.json

```
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Krish",
+    "githubUsername": "krishdevdb",
+    "imageUrl": "https://user-images.githubusercontent.com/76587897/117103511-1db57e80-ad98-11eb-83cc-7dd9ab1f64ab.png",
+    "issueNumber": 473
+  }
```

```suggestion
  },
```
here a comma is missing if you move the coding block at the end you have to add the comma at the top :)